recipes/pro-mdnsd/all/conanfile.py

```diff
@@ -0,0 +1,82 @@
+import os
+import sys
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+from conans.model.version import Version
+
+
+class mdnsdConan(ConanFile):
+    name = "pro-mdnsd"
+    license = "BSD 3-Clause"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    homepage = "https://github.com/Pro/mdnsd"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Improved version of Jeremie Miller's MDNS-SD implementation"
+    topics = ("dns", "daemon", "multicast", "embedded", "c")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "cpp_compatible": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "cpp_compatible": False
+    }
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if not self.options.cpp_compatible:
+            del self.settings.compiler.libcxx
+            del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        if Version(self.version) <= "0.8.0":
+            folder_name = "%s-%s" % ("mdnsd", "0.8")
+        else:
+            folder_name = "%s-%s" % (self.name, self.version)
```

Comment:
```suggestion
        folder_name = self.name + "-" + self.version
```

Reply:
* `self.name` will not work, since the project name is not the same as archive's name 
* `self.version` will not work if I change **conandata.yml** as you suggest in the https://github.com/conan-io/conan-center-index/pull/2009#discussion_r444076645 since the `self.version` will be set to *0.8.0* and archive is set to *0.8*